packages/design/src/SlideTabs/SlideTabs.test.tsx

```diff
@@ -15,30 +15,39 @@
  */
 
 import React from 'react';
+import { screen } from '@testing-library/react';
 
 import { render, fireEvent } from 'design/utils/testing';
 
 import SlideTabs from './SlideTabs';
 
 describe('design/SlideTabs', () => {
   it('renders the supplied number of tabs(3)', () => {
-    const { container } = render(
+    render(
       <SlideTabs
         tabs={['aws', 'automatically', 'manually']}
         onChange={() => {}}
       />
     );
-    expect(container.getElementsByTagName('label')).toHaveLength(3);
+
+    expect(screen.getByLabelText('aws')).toBeInTheDocument();
+    expect(screen.getByLabelText('automatically')).toBeInTheDocument();
+    expect(screen.getByLabelText('manually')).toBeInTheDocument();
```

Comment:
hm, i think this is a different test from the original test? you're testing if these items were rendered, while the original is testing the correct number of expected items. we should keep both?